### ISSUE TITLE
Build Web UI

### ISSUE THREAD

**Issue by samuelowino** [samuelowino/mobile-translate#5]

**Comment by mwangidennis1** [comment]
can i implement the authentication and authorization minus password recovery (do not have details for smtp server)

**Comment by samuelowino** [comment]
Assume it's going to run behind tomcat and yes you can proceed to work on that

**Comment by mwangidennis1** [comment]
It's running on jetty but I can change it if it's required

Le ven. 1 nov. 2024, 08:09, Samuel Owino ***@***.***> a
écrit :

**Comment by mwangidennis1** [comment]
also i plan to bundle freighting on the client side and just give a folder with the right directory structure for translation service

**Comment by samuelowino** [comment]
You are right, that is a client side specific feature. It will be nice if you bundle it separately 
Also ensure the client can still execute translate functions for when there is a desktop app

**Comment by mwangidennis1** [comment]
on it
